models/demos/t3000/falcon40b/tt/falcon_model.py

```diff
@@ -320,7 +320,7 @@ def fwd_prefill(
         )
 
         if self.model_config["LN_INPUT_DTYPE"] != self.model_config["BFP8_DTYPE"]:
-            layer_output = ttnn.experimental.typecast(
+            layer_output = ttnn.typecast(
```

Comment:
this can to stay ttnn.experimental.typecast. it refers to typecast in ttnn/operations/experimental/copy